Tues sprint1 task4 amb

Raotrux opened the pull request:
I've been able to connect to the database and successfully run the GetTasks function. However logic for other functions has yet to be implemented as getting the development environment set up correctly on my PC was difficult and took up the majority of our time.

VitaRox committed:
- Adds in .code-workspace so VS Code workspace config files aren't committed to repo;

VitaRox committed:
- Adds .DS_Store to .gitignore, so these files (which just config the look of the directory windows in the Mac OS UI) aren't committed;

All-Schuck-Up committed:
init

All-Schuck-Up committed:
task 3

Raotrux committed:
Initial commit

Raotrux committed:
Update .gitignore

Raotrux committed:
Update .gitignore

All-Schuck-Up commented:
Is It possible or advisable to add the .venv files to the gitignore?  As I was reviewing I noticed that there are 500+ pip package files.

Raotrux commented:
I added '.venv/' and '.vscode/' to the gitignore but it still committed some of them anyways. I'm sure I did something wrong but I'll look into it and fix that.

Edit:
I added these to my gitignore at the same time the .venv files were being commited. Since they were commited and tracked by github, they weren't removed. I moved my .venv and .vscode files out of the folder, commited the change (it shows you remove the applicable files), and then moved them back in.

Raotrux committed:
Added Task 4 functions

Added GetTask, PutTask, DeleteTask functions using a HTTPTrigger.
Added logic for those functions in dbHandler.py
Adjusted function.json files to properly identify the functions

Raotrux committed:
Removing .venv

Trying to remove the .venv files that made it through before I fixed my gitignore file

Raotrux committed:
Removing .vscode files

Removing .vscode files that made it through before fixing gitignore

Raotrux commented:
Alright, all of my .vscode and .venv files are added to my gitignore and seem to be gone from PR now.

Raotrux committed:
Updated logging code

Added the function name into the line that display logging info so that it displays which function it's running.

RealDogDad reviewed (approved):
Specifying the trigger type?

Raotrux committed:
Updated logic for each HTTPTrigger Function

Updated the logic for GetTask, DeleteTask, PutTask.

Raotrux commented:
> Specifying the trigger type?

What's this in reference to?

Also, I've committed code to my Task 4 functions. GetTask, DeleteTask, PutTask are all functioning for me. So I'd recommend someone else review and approve as well.

jlsnow301 commented on .gitignore:
```diff
@@ -102,3 +102,12 @@ dist
+# VS Code Workspace config files
+*.code-workspace
+
+# Mac window config files
+.DS_Store
+
+.venv/
+.vscode/
```
Oh nice. This closes #49

Raotrux committed:
Updated conn_string

Changed code to include {} for PWD:

PWD={" + PASSWORD + "};

Raotrux committed:
Updated conn_string

Removed the hard coded server to the SERVER environ variable.

Raotrux committed:
Updated gitignore to match current branch

There might have been a better way to do this, I just copied the gitignore file from the current main and pasted it here so the gitignore conflict would go away and it can be merged.

Raotrux committed:
Merge branch 'development' into tues-sprint1-task4-amb

Raotrux commented:
This is ready to merge. I don't know if I'm supposed to do it since I created this PR or if that's the reason I shouldn't be the one to do it. I guess someone let me know if I need to do it.
